src/components/calcite-pick-list/shared-list-logic.ts

```diff
@@ -228,6 +228,16 @@ export function setUpItems<T extends Lists>(
     }
   });
 
+  // remove items from selectedValues if the item doesn't exist in the list
```

Comment:
Can you move this near the end of the method to keep the item-tabbing logic grouped? Bonus points if you extract this into a private method with an expressive name that eliminates the comment. ⭐